README.md

```diff
@@ -111,10 +111,14 @@ export default {
 | `readonly` | boolean | `false` | If field is read only |
 | `placeholder` | string | `` | Native placeholder attribute for input/textarea |
 | `accept` | string |  | Which file types should be accepted if type is file |
-| `validators` | array | `[]` | Which validators should be used to validate input value |
+| `validators` | array | `undefined` | Array holding objects with `validator`{`string`} and `message`{`string`} properties |
```

Comment:
ked je array default daj `[]` alebo ked tak `null` ale `undefined` not cool

Reply:
Resolved in 41869c9